api/appendix_f.asciidoc

```diff
@@ -225,12 +225,12 @@ include::{generated}/api/version-notes/CL_INVALID_PIPE_SIZE.asciidoc[]
 | {CL_INVALID_PLATFORM_anchor}
 
 include::{generated}/api/version-notes/CL_INVALID_PLATFORM.asciidoc[]
-| Returned when the specified platform is not a valid platform.
+| Returned when the specified platform is not a <<valid-object-definition,valid platform>>.
 
 | {CL_INVALID_PROGRAM_anchor}
 
 include::{generated}/api/version-notes/CL_INVALID_PROGRAM.asciidoc[]
-| Returned when a specified program is not a valid program object.
+| Returned when a specified program is not a <<valid-object-definition,valid program object>>.
```

Comment:
This one's a little odd because clGetPlatformInfo and clGetDeviceIDs overload the platform argument to allow NULL, and specify the behavior as implementation-defined. The error description for CL_INVALID_PLATFORM conflicts a bit since it implies you have to return an error for NULL platforms. 
Nothing to hold up this change, but we might want to think about how to resolve that.